.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy to GitHub Pages
+
+on: workflow_dispatch
+
+jobs:
+  deploy:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+          cache: npm
+      - name: Install dependencies
+        run: "cd docs && npm ci"
+      - name: Build
+        run: "cd docs && npm run build"
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: docs/_site
```

Comment:
Yes, this is arbitrary, but since there's now a separation between source files and generated files, figured this is a good change to make.

This is what my GitHub Pages looks like:
<img width="584" alt="image" src="https://user-images.githubusercontent.com/1514176/177842075-743f19fa-3022-4673-85b2-af7ad4797204.png">

Reply:
Oh -- do you have a bare branch for docs?  That's better than our current approach.  I'll likely need some step-by-step help to get that working in this repo.